src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -67,81 +67,73 @@ namespace ZoneEquipmentManager {
 
     struct SimulationOrder
     {
-        // Members
         std::string EquipType;
-        int EquipTypeEnum;
+        int EquipTypeEnum = 0;
         std::string EquipName;
-        int EquipPtr;
-        int CoolingPriority;
-        int HeatingPriority;
-
-        // Default Constructor
-        SimulationOrder() : EquipTypeEnum(0), EquipPtr(0), CoolingPriority(0), HeatingPriority(0)
-        {
-        }
+        int EquipPtr = 0;
+        int CoolingPriority = 0;
+        int HeatingPriority = 0;
     };
 
-    // Functions
-
     void ManageZoneEquipment(EnergyPlusData &state,
-                             bool const FirstHVACIteration,
+                             bool FirstHVACIteration,
                              bool &SimZone, // Set to false at the end of the routine
                              bool &SimAir   // Eventually set to true via SimZoneEquipment if AirLoop must be resimulated
     );
 
     void GetZoneEquipment(EnergyPlusData &state);
 
-    void InitZoneEquipment(EnergyPlusData &state, bool const FirstHVACIteration); // unused 1208
+    void InitZoneEquipment(EnergyPlusData &state, bool FirstHVACIteration); // unused 1208
```

Comment:
Can `FirstHVACIteration` be part of `state` as opposed to a parameter? Asking for a friend.

Reply:
This is obviously not part of this PR, but it's been something that has been bugging me for a while and I just finally blurted it out.